Arrays/Check if all array elements are equal.js

```diff
@@ -0,0 +1,9 @@
+// Write a function that takes an array as argument
+// It should return true if all elements in the array are equal
+// It should return false otherwise
+
+function myFunction(arr) {
+  return arr.every((elm) => elm === arr[0]);
+}
+
+//Problem Link: https://www.jschallenger.com/javascript-arrays/check-if-array-elements-are-equal
```

Comment:
Add Problem link in top. Add a space between comment and //.
Applies for all other places where applicable.

```suggestion
// Problem Link: https://www.jschallenger.com/javascript-arrays/check-if-array-elements-are-equal
```